.github/workflows/release.yml

```
@@ -14,12 +14,12 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up JDK 21
+      - uses: actions/checkout@v4
+      - name: Set up JDK 25
         uses: actions/setup-java@v3
```

actionlint (1.7.10)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)